bin/uno

```diff
@@ -93,6 +93,14 @@ ashell)
   check_dirs
   "$ACCUMULO_HOME"/bin/accumulo shell -u "$ACCUMULO_USER" -p "$ACCUMULO_PASSWORD" "${@:2}"
 	;;
+astart)
+  check_dirs
```

Comment:
This could be a separate issue. I suspect this type of sanity checking would be useful elsewhere. I can open an issue.